### PR TITLE
[Fix] Fix issue in validating ceph module operation

### PR DESCRIPTION
**Pull request by vamahaja**

### CODE REVIEW

**Review by tintumathew10 — COMMENTED** [review]
LGTM

**Comment by vamahaja** [comment]
Testruns -

    http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-yam5z
    http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8v5vg

File: tests/cephadm/test_remove_cluster.py [hunk 1]
```diff
@@ -105,6 +107,11 @@ def run(ceph_cluster, **kw):
                 f"Failed to execute rm-cluster on node '{node.ip_address}'"
             )
 
+    # Check if validation set to `False`
+    if not validation:
```
Comment (pranavprakash20) [inline tests/cephadm/test_remove_cluster.py:111]: can we cross check if the validation variable is str or bool

**Comment by vamahaja** [comment]
Testrun - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-lyd58/

File: tests/cephadm/test_remove_cluster.py (hunk 1 above)
Comment (vamahaja) [reply tests/cephadm/test_remove_cluster.py:111]: Done

**Review by MohitBis — APPROVED** [review]
LGTM

**Pull request merged**